src/components/Header/Header.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const Header = () => {
+  return (
+    <>
+      <h1 className="text-5xl text-grey py-6">Rstrnt Findr-r</h1>
+    </>
```

Comment:
I don't think you need the fragment wrapper here.
Plus the text into the labels file please.